SofaImGui/src/SofaImGui/ImGuiGUIEngine.h

```diff
@@ -29,12 +29,14 @@
 #include <imgui.h>
 #include <sofa/simulation/Node.h>
 #include <SimpleIni.h>
+#include "windows/WindowsManager.h"
```

Comment:
I am not sure you need to include this file in the header. Can you include it only in the cpp file?